tools/cldr-apps/src/test/java/org/unicode/cldr/unittest/web/TestSTFactory.java

```diff
@@ -587,6 +632,7 @@ public void handlePathValue(String path, String value) {
                                     logln(u + " " + elem + "d for " + xpath + " = " + value);
                                 }
                                 break;
+                            case "apiverify": // TODO
```

Comment:
ideally TODO should have the ticket number -- if you're sure to do this as a subsequent PR on this ticket, it would also be good to move new code into subroutines then

Reply:
already done